assets/src/theme/extractPageVariables.js

```diff
@@ -0,0 +1,67 @@
+const balancedVar = require('./balancedVar');
+
+const isSameDomain = ({ href }) => !href || href.indexOf(window.location.origin) === 0;
+const isNotCoreFile = ({ href }) => !href || !href.includes('wp-includes')
```

Comment:
For now I excluded the core files because they have their own variables which won't work for us. Not sure why those stylesheets are loaded :thinking: